dm_cli/bin/dm

```diff
@@ -309,8 +309,8 @@ def entity_cli(context):
     pass
 
 
-@entity_cli.command("import", help="Import the entity (file or package) <source> to the given <destination>.")
-@click.argument("source", required=True, help="Path to file or folder on local filesystem to import. Trailing '/' will result in the content being imported instead of the folder itself")
+@entity_cli.command("import")
+@click.argument("source", required=True)
```

Comment:
click argument does not take the help parameter
https://click.palletsprojects.com/en/8.1.x/documentation/#documenting-arguments